code/modules/surgery/operations/operation_generic.dm

```diff
@@ -0,0 +1,472 @@
+	display_results(
+		surgeon,
+		limb.owner,
+		span_notice("You begin to saw through the bone in [limb.owner]'s [limb.plaintext_zone]..."),
+		span_notice("[surgeon] begins to saw through the bone in [limb.owner]'s [limb.plaintext_zone]."),
+		span_notice("[surgeon] begins to saw through the bone in [limb.owner]'s [limb.plaintext_zone]."),
+	)
+	display_pain(limb.owner, "You feel a horrid ache spread through the inside of your [limb.plaintext_zone]!")
+
+/datum/surgery_operation/limb/saw_bones/on_success(obj/item/bodypart/limb, mob/living/surgeon, obj/item/tool, list/operation_args)
+	. = ..()
+	limb.add_surgical_state(SURGERY_BONE_SAWED)
+	limb.receive_damage(50, sharpness = tool.get_sharpness(), wound_bonus = CANT_WOUND, damage_source = tool)
+	display_results(
+		surgeon,
+		limb.owner,
+		span_notice("You saw [limb.owner]'s [limb.plaintext_zone] open."),
+		span_notice("[surgeon] saws [limb.owner]'s [limb.plaintext_zone] open!"),
+		span_notice("[surgeon] saws [limb.owner]'s [limb.plaintext_zone] open!"),
+	)
+	display_pain(limb.owner, "It feels like something just broke in your [limb.plaintext_zone]!")
+
+/// Fixes sawed bones back together
+/datum/surgery_operation/limb/fix_bones
+	name = "fix bone"
+	desc = "Repair a patient's cut or broken bones."
+	required_bodytype = ~BODYTYPE_ROBOTIC
+	implements = list(
+		/obj/item/stack/medical/bone_gel = 1,
+		/obj/item/stack/sticky_tape/surgical = 1,
+		/obj/item/stack/sticky_tape/super = 2,
+		/obj/item/stack/sticky_tape = 3.33,
+	)
+	preop_sound = list(
+		/obj/item/stack/medical/bone_gel = 'sound/misc/soggy.ogg',
+		/obj/item/stack/sticky_tape/surgical = 'sound/items/duct_tape/duct_tape_rip.ogg',
+		/obj/item/stack/sticky_tape/super = 'sound/items/duct_tape/duct_tape_rip.ogg',
+		/obj/item/stack/sticky_tape = 'sound/items/duct_tape/duct_tape_rip.ogg',
+	)
+	time = 4 SECONDS
+	all_surgery_states_required = SURGERY_SKIN_OPEN
+	any_surgery_states_required = SURGERY_BONE_SAWED|SURGERY_BONE_DRILLED
+
+/datum/surgery_operation/limb/fix_bones/get_default_radial_image()
+	return image(/obj/item/stack/medical/bone_gel)
+
+/datum/surgery_operation/limb/fix_bones/all_required_strings()
+	return ..() + list("the limb must have bones")
+
+/datum/surgery_operation/limb/fix_bones/state_check(obj/item/bodypart/limb)
+	return LIMB_HAS_BONES(limb)
+
+/datum/surgery_operation/limb/fix_bones/on_preop(obj/item/bodypart/limb, mob/living/surgeon, obj/item/tool, list/operation_args)
+	display_results(
+		surgeon,
+		limb.owner,
+		span_notice("You begin to fix the bones in [limb.owner]'s [limb.plaintext_zone]..."),
+		span_notice("[surgeon] begins to fix the bones in [limb.owner]'s [limb.plaintext_zone]."),
+		span_notice("[surgeon] begins to fix the bones in [limb.owner]'s [limb.plaintext_zone]."),
+	)
+	display_pain(limb.owner, "You feel a grinding sensation in your [limb.plaintext_zone] as the bones is being set back in place.")
```

Comment:
```suggestion
	display_pain(limb.owner, "You feel a grinding sensation in your [limb.plaintext_zone] as the bones are set back in place.")
```
this is not the bone setting operation but i couldnt think of a more suitable message